lib/cfa/grub2/default.rb

```diff
@@ -77,7 +77,7 @@ def xen_hypervisor_params
 
       def xen_kernel_params
         @xen_kernel_params ||= KernelParams.new(
-          data["GRUB_CMDLINE_LINUX_XEN"], "GRUB_CMDLINE_LINUX_XEN"
+          data["GRUB_CMDLINE_XEN_DEFAULT"], "GRUB_CMDLINE_XEN_DEFAULT"
```

Comment:
A link for documentation about that option, well about Xen best practices in general: https://wiki.xenproject.org/wiki/Xen_Project_Best_Practices